protocol/synthetix/cannonfile.toml

```diff
@@ -158,8 +170,8 @@ fromCall.func = "owner"
 func = "initOrUpgradeToken"
 args = [
     "0x555344546f6b656e000000000000000000000000000000000000000000000000",
-    "Synthetic USD Token v3",
-    "snxUSD",
+    "<%= settings.usd_token_name %>",
```

Comment:
![image](https://github.com/user-attachments/assets/d1578acd-28c7-49c5-a75a-bc86281edc4d)
we can already do it I guess.
Also can you replace `0x555344546f6b656e000000000000000000000000000000000000000000000000` (and account token too) with this please

```
[setting.usd_token_name]
defaultValue = "<%= formatBytes32String('USDToken') %>"
```

Reply:
yea true, but we should really be doing it through the official function. so I guess we can call this cleanup to bring upstream as we go

will do the replacements